src/services/postDetail.ts

```
@@ -0,0 +1,454 @@
+        query: PostDetailGQL.QUERIES.GET_POST_LIST_LIGHT,
+        variables: { start },
+        fetchPolicy: 'no-cache'
+      });
+      return data?.getPostList ?? [];
+    } catch (error) {
+      console.error('게시물 목록 조회 오류:', error);
+      throw error;
+    }
+  },
+
+  getPostById: async (client: ApolloClient<any>, id: string): Promise<PostDetailUnion | null> => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)